src/locale/locales/fr/messages.po

```diff
@@ -2505,6 +2567,10 @@ msgstr "Voir le profil du compte"
 msgid "Graphic Media"
 msgstr "Médias crus"
 
+#: src/state/shell/progress-guide.tsx:167
+msgid "Half way there!"
+msgstr "A mi-chemin !"
```

Comment:
It's _technically_ a good translation, but I think that translating it more like `nearly there` is more natural in that context.

```suggestion
msgstr "On y est presque !"
```